app/components/downloaders/administer_access_component.html.erb

```diff
@@ -1,16 +1,11 @@
 <h4 class="mt-4"><%= t('downloaders.administer_access_component.heading') %></h4>
-<% unless @organization.restrict_downloads %>
-  <div class="alert alert-warning" role="alert">
-    <%= t('downloaders.administer_access_component.unrestricted_alert') %>
-  </div>
-<% end %>
```

Comment:
I'm assuming you've intentionally dropping this banner because the icons accurately reflect the actual state?

Reply:
Yup. And I'm about to put up another PR with a more helpful message.